build/common-oauth.props

```diff
@@ -0,0 +1,22 @@
+<Project>
+    <Import Project=".\common.props" />
+
+    <PropertyGroup>
+        <OutputType>Library</OutputType>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <EmbeddedResource Include="compiler\resources\**\*" />
+    </ItemGroup>
+
+    <ItemGroup>
+        <ProjectReference Include="../AspNet.Security.OAuth.Extensions/AspNet.Security.OAuth.Extensions.csproj" />
+        <PackageReference Include="JetBrains.Annotations" Version="10.2.1" PrivateAssets="All" />
```

Comment:
`10.3.0`.

That would be cool if you could use the same pattern as the one I used here: https://github.com/aspnet-contrib/AspNet.Security.OAuth.Extensions/pull/63/files#diff-30dcc9789f8ea3c7632e4f054e60c072